x/inflation/types/v1beta2/params.go

```diff
@@ -0,0 +1,96 @@
+package v1beta2
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/pkg/errors"
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+const (
+	DefaultInflationDecayFactor uint32 = 2 // years
+
+	keyInflationDecayFactor = "InflationDecayFactor"
+	keyInitialInflation     = "InitialInflation"
+	keyVariance             = "Variance"
+)
+
+var (
+	DefaultInitialInflation = sdk.NewDec(100)
+	DefaultVarince          = sdk.MustNewDecFromStr("0.05")
+
+	MaxInitialInflation = sdk.NewDec(100)
+	MinInitialInflation = sdk.ZeroDec()
+
+	MaxVariance = sdk.NewDec(1)
+	MinVariance = sdk.ZeroDec()
```

Comment:
Make these functions so that they're constant-like.

Reply:
Done